lib/inventory_refresh/save_collection/saver/sql_helper.rb

```diff
@@ -116,7 +116,7 @@ def complement_of!(manager_uuids)
         all_entities     = Arel::Table.new(:all_entities)
         all_entities_cte = Arel::Nodes::As.new(
           all_entities,
-          Arel.sql("(#{inventory_collection.full_collection_for_comparison.select(:id, *all_attribute_keys_array).to_sql})")
+          Arel.sql("(#{inventory_collection.full_collection_for_comparison.active.select(:id, *all_attribute_keys_array).to_sql})")
```

Comment:
This assumes there is an active scope on the association?  Seems like embedding app logic into the gem.  Can we add an assoc like `active_container_abcd` in topological_inventory-core and use that instead?

Reply:
I can probably tie this to the ```:retention_strategy => :archive``` that should tell us if it's archivable. Then it's more of a 'reserved columns' than embedding specific logic.

Reply:
I can probably tie this to the ```:retention_strategy => :archive``` that should tell us if it's archivable. Then it's more of a 'reserved columns' than embedding specific logic.